### ISSUE TITLE
Add D2D renderer

### ISSUE THREAD

**Issue by be5invis** [adobe/svg-native-viewer#61]

**Comment by mpsuzuki** [comment]
Indeed, we should add D2D port. Dirk had once worked for [WinDrawLib](https://github.com/mity/windrawlib) port, I guess it was a next step after GDI+ port. I want to hear his comment whether he found any critical problem and abandoned this project, or he moved to another port calling D2D directory.

BTW, I'm really newbie about D2D programming. Is there any nice tutorial?

**Comment by dirkschulze** [comment]
@mpsuzuki If you used GDI+ before it is not hard to get into D2D. The basic idea of rendering is similar.
Would you be interested in working on the D2D port? I could try to get someone in my team interested but that will take a couple of weeks at least.

**Comment by mpsuzuki** [comment]
I think the request of the migration to D2D is reasonable, but I've never experienced with GDI+ programming. Maybe a couple of weeks is to short for me to learn D2D. I want to hear whether any experts from your team would be interested in this issue. If nobody would work (that's really OK, no problem), and there is no technical barrier, I should work for this.

**Comment by dirkschulze** [comment]
@mpsuzuki No problem :)

**Comment by dirkschulze** [comment]
An early prototype can be found here: https://github.com/dirkschulze/svg-native-viewer/tree/d2d-port (crashing after drawing  currently).
